controllers/src/test/scala/cromwell/pipeline/controller/ProjectControllerTest.scala

```diff
@@ -40,5 +39,64 @@ class ProjectControllerTest
         }
       }
     }
+
+    "delete project by id" should {
+      "return deactivated project entity" in {
+        val dummyProject = TestProjectUtils.getDummyProject()
+        val accessToken = AccessTokenContent(TestUserUtils.getDummyUser().userId.value)
+        val deactivatedProject = dummyProject.copy(active = false)
+        val request = ProjectDeleteRequest(dummyProject.projectId)
+
+        when(projectService.deactivateProjectById(dummyProject.projectId, UserId(accessToken.userId)))
+          .thenReturn(Future.successful(Some(deactivatedProject)))
+
+        Delete("/projects", request) ~> projectController.route(accessToken) ~> check {
+          responseAs[Project] shouldBe deactivatedProject
+        }
+
+      }
+
+      "return server error if project deactivation was failed" taggedAs (Controller) in {
+        val userId = TestUserUtils.getDummyUser().userId
+        val request = ProjectDeleteRequest(TestProjectUtils.getDummyProject().projectId)
+        val accessToken = AccessTokenContent(userId.value)
+
+        when(projectService.deactivateProjectById(request.projectId, userId))
+          .thenReturn(Future.failed(new RuntimeException("Something wrong")))
+
+        Delete("/projects", request) ~> projectController.route(accessToken) ~> check {
+          status shouldBe StatusCodes.InternalServerError
+        }
+      }
+    }
+
+    "update project" should {
+      "return status code NoContent if " in {
+        val userId = TestUserUtils.getDummyUser().userId
+        val accessToken = AccessTokenContent(userId.value)
+        val dummyProject = TestProjectUtils.getDummyProject()
+        val request = ProjectUpdateRequest(dummyProject.projectId, dummyProject.name, dummyProject.repository)
+
+        when(projectService.updateProject(request, userId)).thenReturn(Future.successful(1))
+
+        Put("/projects", request) ~> projectController.route(accessToken) ~> check {
+          status shouldBe StatusCodes.NoContent
+        }
+      }
+
+      "return InternalServerError status if project id doesn't match" in {
```

Comment:
How does this test checks that the project id doesn't match?

Reply:
fixed

Reply:
Nothing has changed. Therefore, nothing is fixed.

Reply:
Sorry, I didn't push. Fixed now